src/commands/interviews/interviewers.ts

```diff
@@ -40,8 +40,7 @@ class InterviewersCommand extends BaseCommand {
   }
 
   private async getInterviewerDisplayInfo(interviewer: Interviewer) {
-    const userTag = (await this.client.users.fetch(interviewer['user_id'])).tag;
-    return '**' + userTag + '** | [Calendar](' + interviewer['link'] + ')\n\n';
+    return `${await this.client.users.fetch(interviewer['user_id'])} | [Calendar](${interviewer['link']})\n\n`;
```

Comment:
nit: can we take `await this.client.users.fetch(interviewer['user_id'])` out of the template string and store in a variable like before? I feel like it'd be more readable than having everything in 1 line.

Reply:
Changed.